src/version.rc

```diff
@@ -0,0 +1,113 @@
+// -rick
+//
+// Microsoft Visual C++ generated resource script.
+//
+#include "resource.h"
+#ifdef __GNUC__ // make windows rc accept this
+#include <winresrc.h>
+#endif
+/////////////////////////////////////////////////////////////////////////////
+// English (United States) resources
+
+#define STRINGIZER(version) #version
+
+#define VERSION_STRING(version, codename, revision) \
+  STRINGIZER(version) "-release [" STRINGIZER(codename) "] (rev-" STRINGIZER(revision) ")"
+
+#if !defined(AFX_RESOURCE_DLL) || defined(AFX_TARG_ENU)
+#ifdef _WIN32
+LANGUAGE 1033,1
+#pragma code_page(1252)
+#endif //_WIN32
+
+#ifdef APSTUDIO_INVOKED
+/////////////////////////////////////////////////////////////////////////////
+//
+// TEXTINCLUDE
+//
+
+1 TEXTINCLUDE
+BEGIN
+    "resource.h\0"
+END
+
+2 TEXTINCLUDE
+BEGIN
+    "\0"
+END
+
+3 TEXTINCLUDE
+BEGIN
+    "\r\n"
+    "\0"
+END
+
+#endif    // APSTUDIO_INVOKED
+
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// Version
+//
+
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION LLARP_VERSION
+ PRODUCTVERSION LLARP_VERSION
+ FILEFLAGSMASK 0x17L
+#ifdef _DEBUG
+ FILEFLAGS 0x3L
+#else
+ FILEFLAGS 0x2L
+#endif
+ FILEOS 0x40004L
+ FILETYPE 0x1L
+ FILESUBTYPE 0x0L
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "Comments", "includes relay/exit functionality, such code is highly experimental on non-Linux targets"
+            VALUE "CompanyName", "Loki Foundation"
+            VALUE "FileDescription", "LokiNET for Microsoft� Windows� NT�"
+            VALUE "FileVersion", "0.8.0 (proof of soon)" //VERSION_STRING(LLARP_VERSION_TRIPLET, LLARP_RELEASE_MOTTO, VERSIONTAG)
+            VALUE "InternalName", "llarpd"
+            VALUE "LegalCopyright", "Copyright �2018-2020 Jeff Becker, Rick V for the Loki Foundation. All rights reserved. This software is provided under the terms of the zlib-libpng licence; see the file LICENSE for details."
+            VALUE "OriginalFilename", "llarpd.exe"
+            VALUE "ProductName", "LokiNET for Windows"
+            VALUE "ProductVersion", "0.8.0 (proof of soon)" //VERSION_STRING(LLARP_VERSION_TRIPLET, LLARP_RELEASE_MOTTO, VERSIONTAG)
+        END
+    END
+    BLOCK "VarFileInfo"
+    BEGIN
+        VALUE "Translation", 0x409, 1200
+    END
+END
+
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// RT_MANIFEST
+//
+
+// Uncomment if your toolchain does not provide a default-manifest.o
+//3                       RT_MANIFEST             "app.xml"
+
+#endif    // English (United States) resources
+/////////////////////////////////////////////////////////////////////////////
+
+
+
+#ifndef APSTUDIO_INVOKED
+/////////////////////////////////////////////////////////////////////////////
+//
+// Generated from the TEXTINCLUDE 3 resource.
+//
+
+
+/////////////////////////////////////////////////////////////////////////////
+#endif    // not APSTUDIO_INVOKED
```

Comment:
JFC Windows.  We get both XML *AND* this archaic stuff?  Joy.

Reply:
ha
it's what tells the runtime link editor to pop up a UAC prompt so we can do exits

Reply:
if you look over......